### PR TITLE
Rocklib

### PR DESCRIPTION
**Pull request by JustinMoss**

### CODE REVIEW

File: Rock.Encryption/Symmetric/RNG.cs [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
+namespace RockLib.Encryption.Symmetric
+#else
 namespace Rock.Encryption.Symmetric
+# endif
 {
     internal static class RNG
     {
-        private static readonly ThreadLocal<RNGCryptoServiceProvider> _instance =
-            new ThreadLocal<RNGCryptoServiceProvider>(() => new RNGCryptoServiceProvider());
+        private static readonly ThreadLocal<RandomNumberGenerator> _instance =
```
Comment (derikwhittaker) [inline Rock.Encryption/Symmetric/RNG.cs]: why the change in crypto provider/generator

**Comment by derikwhittaker** [comment]
Appears pretty straight forward.  Most the code was #if blocks for the different frameworks.  Just 1 comment/feedback/

File: Rock.Encryption/Symmetric/RNG.cs (hunk 1 above)
Comment (JustinMoss) [inline Rock.Encryption/Symmetric/RNG.cs]: RNGCrypto wasn't available in standard, but result was the same with straight RandomNumberGenerator.

Comment (derikwhittaker) [inline Rock.Encryption/Symmetric/RNG.cs]: Good enough for me.
